sympy/solvers/solveset.py

```diff
@@ -1136,8 +1136,8 @@ def _solve_exponential(lhs, rhs, symbol, domain):
 
 def _is_exponential(f, symbol):
     r"""
-    Return True if one or more terms contain symbol only in exponents,
```

Comment:
Please keep the command-form of the docstring [per PEP 257](https://www.python.org/dev/peps/pep-0257/); so `Return`, not `Returns`.